alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java

```diff
@@ -217,8 +231,16 @@ public void testTransformation()
             sourceFile, sourceMimetype, targetMimetype, targetExtension);
         try
         {
+            // note: some image/tiff->image/* will return multiple page results (hence error) unless options specified for single page
+            Map<String, String> tOptions = emptyMap();
+            Pair targetPair = Pair.of(targetExtension, targetMimetype);
+            if (targetExtensionsForTiffFirstPage.contains(targetPair))
```

Comment:
From a quick look, now we add those options to all types of source files, not just .tiff?

Reply:
Ah, yes - nice catch. I'lll make it more explicit for this special case.

Reply:
Updated, thanks.

Reply:
LGTM after the update, thanks.